pyproject.toml

```diff
@@ -33,6 +33,7 @@ scikit-learn = "^1.0"
 category_encoders = "^2.2.2"
 scanpy = "^1.8.1"
 mudata = "^0.1.0"
+leidenalg = "^0.8.7"
```

Comment:
why is this added?

Reply:
Leidenalg is an optional dependency in Scanpy. However, we need it for clustering as is now included in our example notebook. It should imo always be installed.